index.rst

```diff
@@ -282,22 +290,20 @@ Prompt processing needs a way to identify which visits to trigger prompt process
 using the 'nextVisit' event captured in lsst.sal.ScriptQueue.logevent_nextVisit.
 Currently the only metadata key propagated to nextVisit is a "survey" key,
 which corresponds to the science_program.
+Potentially, the contents of the "survey" key could be modified separately from the science_program value,
+by (for example) adding observation_reason or target_name.
 PP can create a list of science_programs which will trigger prompt processing.
 
 
 Calibration
 -----------
 
 Calibration processing needs a way to unique identify data within a single "run" of a calibration process.
-This process will ideally be run by the FBS,
-but will need to use several different JSON BLOCKs to acquire exposures using different scripts.
-The science_program in the JSON BLOCKs could be overridden to match a single value if desirable.
-The "run" identifier is not currently available and cannot easily be set without some additional machinery.
-One option would be to track an incremental run ID in the FBS and store this in the observation_reason
-or possibly scheduler_note.
-A potential drawback might be if science_program is overriden in the BLOCK to be a single value for all
-contributing content for the 'run', and observation_reason is only the incremental run id,
-then the trace to the JSON BLOCK in use for a given observation could be lost.
+The group_id in the JSON BLOCKs could be written as a simple string which would match for the entire block,
```

Comment:
This isn't quite right in that we don't allow a user-input string for the group_id. Another way to say it is:

"There is a feature unique to the calibration data taking scripts that allows the group_id for all images taken together in a single JSON BLOCK to be set to match the unique observation id that is assigned to the JSON BLOCK execution. This observation id is generated by the image naming service, and has the format `BT220_O_20241024_000001`, which corresponds to the first execution 001 of BLOCK-T220 on 2024-10-24." This allows users to uniquely identify calibration runs.

Reply:
I didn't mean to imply it would be a user-input string, but rather as you say - a simple string which is the BLOCK-ID-DAYOBS-run# .. 
but your more complete explanation is better anyway so i'll sub that in.